roles/kubernetes/preinstall/vars/fedora.yml

```diff
@@ -1,5 +1,6 @@
 ---
 required_pkgs:
-  - libselinux-python3
+  # - libselinux-python3
+  - python3-libselinux
```

Comment:
The package name libselinux-python3 seems to be wrong with fedora 31 coreos (it is wrong with fedora 32 as well). It must be python3-libselinux. However, I have no idea when fedora changed the package name.

Reply:
I checked this on F30 and F31, `dnf install libselinux-python3` installs `python3-libselinux`

Like:
```console
dnf install libselinux-python3
Last metadata expiration check: 0:01:01 ago on Sun Apr 19 13:17:57 2020.
Dependencies resolved.
===========================================================================================
 Package                     Architecture    Version                Repository        Size
===========================================================================================
Installing:
 python3-libselinux          x86_64          2.9-3.1.fc30           updates          162 k

Transaction Summary
===========================================================================================
Install  1 Package

Total download size: 162 k
Installed size: 635 k
Is this ok [y/N]:
```

So good to use the canonical package name. 👍

Reply:
The change set now renames _all_ occurences of `libselinux-python3` to `python3-libselinux`.